pyscriptjs/src/plugin.ts

```diff
@@ -39,6 +39,10 @@ export class Plugin {
      */
     afterSetup(runtime: Runtime) {}
 
+    beforePyScriptExec(runtime, src, PyScriptTag) {}
```

Comment:
This new life cycle is pretty self explanatory, but could we add docstring to both similar to what Antonio did on the others ones? 🤔

Reply:
Agreed, will do.